### PR TITLE
First attempt at registering new AP40 Pelican origin (INF-1796)

### PR DESCRIPTION
**Pull request by brianhlin**
This is going to be a very odd registration: we currently have the production AP running on `ap40.uw.osg-htc.org`, aka `ospool-2040.chtc.wisc.edu`. However, we need to stand up a new origin on decent hardware as `origin-auth2001` is falling over under load. We are using the new AP40 hardware, `ospool-2140.chtc.wisc.edu` for this in a similar setup to the UC APs (we plan to do an in-place replacement of the old AP40 with the new AP40).

I'm not sure what the right registrations are here exactly:
1.  We want to serve `/ospool/PROTECTED` from the new origin to address the acute issue
1.  But we also want to move users over to `/ospool/ap40/data` with the future AP40 transition
I'm not entirely sure what the issuers should be and the relevant FQDNs across the different fields.

### CODE REVIEW

File: virtual-organizations/OSG.yaml [hunk 1]
```diff
@@ -256,6 +257,23 @@ DataFederations:
...
+              Map Subject: True
+        AllowedOrigins:
+          - CHTC-ap40
+        AllowedCaches:
+          - ANY
+        Writeback: https://ap40.uw.osg-htc.org:8443
+        DirList: https://ap40.uw.osg-htc.org:8443
+        CredentialGeneration:
+          Strategy: OAuth2
+          Issuer: https://osg-htc.org/ospool
```
Comment (matyasselmeci) [inline virtual-organizations/OSG.yaml:274]:
```suggestion
          Issuer: https://ap40.uw.osg-htc.org:8443
```
?

File: virtual-organizations/OSG.yaml [hunk 2]
```diff
@@ -136,10 +136,11 @@ DataFederations:
               Map Subject: True
         AllowedOrigins:
           - CHTC_OSPOOL_ORIGIN
+          - CHTC-ap40
         AllowedCaches:
           - ANY
-        Writeback: https://origin-auth2001.chtc.wisc.edu:1095
-        DirList: https://origin-auth2001.chtc.wisc.edu:1095
+        Writeback: https://ospool-ap2140.chtc.wisc.edu:8443
+        DirList: https://ospool-ap2140.chtc.wisc.edu:8443
```
Comment (matyasselmeci) [inline virtual-organizations/OSG.yaml:143]: You should leave this for a separate PR: we'll need to merge this one to be able to generate the authfile/scitokens config for ap2140 but we shouldn't change the Writeback and DirList until we're ready to switch everything over.

Comment (brianhlin) [reply virtual-organizations/OSG.yaml]: We're going to pull this entirely since we're going to need a flag day

File: topology/University of Wisconsin/CHTC/CHTC.yaml [hunk 3]
```diff
@@ -526,15 +526,23 @@ Resources:
...
+    DN: /CN=ospool-ap2140.chtc.wisc.edu
     Services:
       Submit Node:
         Description: OS Pool access point
         Details:
           hidden: false
+      XRootD origin server:
+        Description: OSDF Pelican origin
+        Details:
+          hidden: false
```
Comment (matyasselmeci) [inline topology/University of Wisconsin/CHTC/CHTC.yaml:539]:
```suggestion
          hidden: false
          endpoint_override: ospool-ap2140.chtc.wisc.edu:8443
          auth_endpoint_override: ospool-ap2140.chtc.wisc.edu:8443          
```

**Review by matyasselmeci — APPROVED** [review]
LGTM. Merge at your convenience.
